client/app/scripts/superdesk-publish/publish.js

```diff
@@ -184,7 +184,7 @@
         function fetchPublishQueue () {
             var criteria = criteria || {};
             criteria.max_results = 200;
-            criteria.sort = '[(\'_created\',-1)]';
+            criteria.sort = '[(\'published_seq_num\',-1)]';
```

Comment:
Perhaps just use double quotes to avoid the need for escaping, makes it a bit easier on the eyes, but it's a minor thing.